.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Screenshots**
+Include screenshots that will clarify your issue and/or solution
+
+**Links to blueprints, solution yamls, terraform templates**
+If applicable, link to public examples that can be used to reproduce your issue. repos, catalog entries, etc
```

Comment:
Replace the last four words with:
For example, repos, catalog entries, and so on.